src/test/scala/is/hail/testUtils/RichTable.scala

```diff
@@ -34,4 +38,98 @@ class RichTable(ht: Table) {
     }
   }
 
+  def rename(rowUpdateMap: Map[String, String], globalUpdateMap: Map[String, String]): Table = {
+    select(ht.fieldNames.map(n => s"${ rowUpdateMap.getOrElse(n, n) } = row.$n"): _*)
+      .keyBy(ht.key.map(k => rowUpdateMap.getOrElse(k, k)))
+      .selectGlobal(ht.globalSignature.fieldNames.map(n => s"${ globalUpdateMap.getOrElse(n, n) } = global.$n"): _*)
+  }
+
+  def select(exprs: Array[String]): Table = {
+    val ec = ht.rowEvalContext()
+
+    val (paths, asts) = Parser.parseSelectExprsToAST(exprs, ec)
```

Comment:
To be honest, I might not bother with handling the IR path in the test code.  That means you don't need to add a another `parse...` method.  (Ripping those out is going to make me so happy.)